cluster/kubectl.sh

```diff
@@ -18,6 +18,14 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+echo "-=-=-=-=-=-=-=-=-=-="
+echo "NOTE:"
+echo "kubectl.sh is deprecated and will be removed soon."
+echo "please replace all usage with calls to the kubectl"
+echo "binary and ensure that it is in your PATH." 
```

Comment:
Looks like kubectl.sh, in addition to looking for the kubectl binary, also sets --context for GKE and --server for ubuntu. Do we want to warn about this as well or is this no longer necessary?

If it's still necessary and that particular wrapper is needed, maybe we can have a separate script or somehow bake this into the providers (or just warn).